README.md

```diff
@@ -40,7 +40,7 @@ Clone the repo and extract it
 Install the project dependencies  
 `cd Probability-Programs`  
 `pipenv install`  
-Make sure you've the `"%AppData%\Python\Python3X\Scripts"` directory added to PATH env variable.
+`Make sure the `"%AppData%\Python\Python3X\Scripts" `directory added to PATH env variable.`
```

Comment:
Sorry I made a mistake on the formatting of the review comment.  
It should actually be  
Make sure the `"%AppData%\Python\Python3X\Scripts" `directory is added to PATH env variable.  
You forgot the is between directory and added

Reply:
Nvm, I was about to ask you why you'd put everything except the dir into code